### ISSUE TITLE
First example on documentation doesn't work due to density plot

### ISSUE THREAD

**Issue by DoktorMike** [TuringLang/Turing.jl#640]
The following code from the first example at http://turing.ml/docs/get-started/ doesn't work for me.

```
using Turing
using Plots

# Define a simple Normal model with unknown mean and variance.
@model gdemo(x, y) = begin
  s ~ InverseGamma(2,3)
  m ~ Normal(0,sqrt(s))
  x ~ Normal(m, sqrt(s))
  y ~ Normal(m, sqrt(s))
  return s, m
end

#  Run sampler, collect results
c1 = sample(gdemo(1.5, 2), SMC(1000))
c2 = sample(gdemo(1.5, 2), PG(10,1000))
c3 = sample(gdemo(1.5, 2), HMC(1000, 0.1, 5))
c4 = sample(gdemo(1.5, 2), Gibbs(1000, PG(10, 2, :m), HMC(2, 0.1, 5, :s)))
c5 = sample(gdemo(1.5, 2), HMCDA(1000, 0.15, 0.65))
c6 = sample(gdemo(1.5, 2), NUTS(1000,  0.65))

# Summarise results (currently requires the master branch from MCMCChain)
describe(c3)

# Plot and save results
p = plot(c3)
savefig("gdemo-plot.png")
```

The error is:
> julia> # Plot and save results
       p = plot(c3)
┌ Warning: seriestype density has been moved to StatPlots.  To use: `Pkg.add("StatPlots"); using StatPlots`
└ @ Plots ~/.julia/packages/Plots/PGn1u/src/args.jl:1054
┌ Warning: seriestype density has been moved to StatPlots.  To use: `Pkg.add("StatPlots"); using StatPlots`
└ @ Plots ~/.julia/packages/Plots/PGn1u/src/args.jl:1054
┌ Warning: seriestype density has been moved to StatPlots.  To use: `Pkg.add("StatPlots"); using StatPlots`
└ @ Plots ~/.julia/packages/Plots/PGn1u/src/args.jl:1054
┌ Warning: seriestype density has been moved to StatPlots.  To use: `Pkg.add("StatPlots"); using StatPlots`
└ @ Plots ~/.julia/packages/Plots/PGn1u/src/args.jl:1054
┌ Warning: seriestype density has been moved to StatPlots.  To use: `Pkg.add("StatPlots"); using StatPlots`
└ @ Plots ~/.julia/packages/Plots/PGn1u/src/args.jl:1054
┌ Warning: seriestype density has been moved to StatPlots.  To use: `Pkg.add("StatPlots"); using StatPlots`
└ @ Plots ~/.julia/packages/Plots/PGn1u/src/args.jl:1054
ERROR: The backend must not support the series type Val{:density}, and there isn't a series recipe defined.
Stacktrace:
 [1] error(::String) at ./error.jl:33
 [2] apply_recipe(::Dict{Symbol,Any}, ::Type{Val{:density}}, ::UnitRange{Int64}, ::Array{Float64,1}, ::Nothing) at /home/michael/.julia/packages/RecipesBase/Uz5AO/src/RecipesBase.jl:275
 [3] _process_seriesrecipe(::Plots.Plot{Plots.GRBackend}, ::Dict{Symbol,Any}) at /home/michael/.julia/packages/Plots/PGn1u/src/pipeline.jl:408
 [4] _plot!(::Plots.Plot{Plots.GRBackend}, ::Dict{Symbol,Any}, ::Tuple{Turing.Chain{AbstractRange{Int64}}}) at /home/michael/.julia/packages/Plots/PGn1u/src/plot.jl:234
 [5] #plot#132(::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Function, ::Turing.Chain{AbstractRange{Int64}}) at /home/michael/.julia/packages/Plots/PGn1u/src/plot.jl:57
 [6] plot(::Turing.Chain{AbstractRange{Int64}}) at /home/michael/.julia/packages/Plots/PGn1u/src/plot.jl:51
 [7] top-level scope at none:0

**Comment by xukai92** [comment]
Did you try to `Pkg.add("StatPlots")`?

**Comment by DoktorMike** [comment]
> Did you try to `Pkg.add("StatPlots")`?

Yes indeed I already have StatPlots installed. And changing the line in the top from

```using Plots```

to 

```using StatPlots```

Fixes the issue. So I believe the example should be updated.
